evalai/utils/submissions.py

```diff
@@ -0,0 +1,63 @@
+import requests
+import sys
+
+from beautifultable import BeautifulTable
+from click import echo, style
+from datetime import datetime
+
+from evalai.utils.auth import get_request_header
+from evalai.utils.common import validate_token
+from evalai.utils.urls import URLS
+from evalai.utils.config import API_HOST_URL, EVALAI_ERROR_CODES
+
+
+def pretty_print_my_submissions_data(submissions):
+    """
+    Funcion to print the submissions for a particular Challenge.
+    """
+    table = BeautifulTable()
+    attributes = ["id", "participant_team_name", "execution_time", "status"]
+    columns_attributes = ["ID", "Participant Team", "Execution Time(sec)", "Status", "Submitted At", "Method Name"]
+    table.column_headers = columns_attributes
+
+    for submission in submissions:
+        # Format date
+        date = datetime.strptime(submission['submitted_at'], "%Y-%m-%dT%H:%M:%S.%fZ")
+        date = date.strftime('%A %d %B %Y')
```

Comment:
`date = date.strftime('%D %r')`
@isht3 One more thing that is to be figured out is that the time returned from API is in `UTC` format and we can't display that directly to a user as it will be confusing.

Reply:
@RishabhJain2018 Should it be displayed (UTC) on the attribute header just for the time being? I'll start an issue to add this feature. Is it okay?